### DISCUSSION TITLE
What does the Uno platform rely on from the Microsoft ecosystem (or others) ?

### DISCUSSION THREAD

**Discussion by stephenhauck** [unoplatform/uno#15452] [Q&A] [answered]
I wanted to understand what the Uno platform depends on before I FULLY commit.
Is Uno reliant on anything MAUI at all ?
Is Uno reliant on any MAUI extension like Essentials NuGet ?
What does Uno depend on from external vendors .. SDK's and such ?
How much does paid support cost and is it very different than this area ?

Thanks

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the questions.

> Is Uno reliant on anything MAUI at all ?

Uno relies on .NET iOS/Android/Catalyst workloads, .NET for WebAssembly through Uno.Wasm.Bootstrap and .NET for the rest. The UI layer of MAUI is not used by the core of Uno Platform. The only link to MAUI is when using [Uno Extensions MAUI Embedding](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Maui/MauiOverview.html), with the scenario of using MAUI controls in an Uno app.

> Is Uno reliant on any MAUI extension like Essentials NuGet ?

No. Uno Platform contains its own implementation of WinRT APIs (which have similar APIs to Essentials). Still, MAUI Essentials can optionally be used on platforms that are supported by MAUI.

> What does Uno depend on from external vendors .. SDK's and such ?

It depends on which level, but for the point of view of a developer using Uno Platform at build time, dependencies are Nuget.org, .NET SDKs and the Uno.Wasm.Bootstrap dependency for the WebAssembly runtime that is currently hosted by Uno Platform. We intend to move to nuget.org for that last part as well.

For mobile platforms, recent Android SDKs and Apple Xcode are needed.

> How much does paid support cost and is it very different than this area ?

I'll let @sasakrsmanovic chime in.

**Reply by stephenhauck** [reply]
Thanks for the detailed response

**Reply by sasakrsmanovic** [reply]
Thanks. We support community for free, on GitHub discussions and at our new discord server https://platform.uno/uno-discord . I highly recommend going down the free/community route; it is very lively and responsive. Talk is cheap, so give it a shot. Don't believe me but your own experiences. 

That said, for clients for whom it is necessary to have paid support, be it because of corporate policies or they need SLAs exceeding what they get for free in the community we have paid support plans, which is how we support Uno Platform https://platform.uno/blog/sustaining-the-open-source-uno-platform/ , and it entitles clients with private GH boards and discussion boards, among other things. Pricing varies depending on many factors including involvement of individual/company with our open source project, geography etc. The best thing to do is to poke around, try a POC, try the platform and try the free community support. Hope to see you as part of our community Stephen!
